dashboard/src/shared/Operators.ts

```diff
@@ -1,10 +1,11 @@
 import * as urls from "../shared/url";
 import { axiosWithAuth } from "./AxiosInstance";
+import { definedNamespaces } from "./Namespace";
 import { IClusterServiceVersion, IK8sList, IPackageManifest, IResource } from "./types";
 
 export class Operators {
   public static async isOLMInstalled() {
-    const { status } = await axiosWithAuth.get(urls.api.operators.crd);
+    const { status } = await axiosWithAuth.get(urls.api.operators.operators(definedNamespaces.all));
```

Comment:
Does this work if you don't have RBAC for all namespaces?

Reply:
well, no, typically the admin should have cluster-wide privileges but it's true that it's not necessary for this